onnxruntime/core/providers/cpu/cpu_provider_factory.cc

```diff
@@ -15,7 +15,7 @@ struct CpuProviderFactory {
   CpuProviderFactory();
 };
 
-ONNXStatus* ONNXRUNTIME_API_STATUSCALL CreateCpu(void* this_, ONNXRuntimeProvider** out) {
+ONNXStatus* ONNXRUNTIME_API_CALL CreateCpu(void* this_, ONNXRuntimeProvider** out) {
```

Comment:
For all the function implementations, please change it to 
ONNXRUNTIME_API(...)
or 
ONNXRUNTIME_API_STATUS_IMPL(...)